src/app/_hooks/useSetParams.js

```diff
@@ -0,0 +1,23 @@
+import { useRouter, useSearchParams } from 'next/navigation';
+
+export function useSetParam(param) {
+  const searchParams = useSearchParams();
+  const router = useRouter();
+  const newParams = new URLSearchParams(searchParams);
+  const addParam = value => {
+    if (searchParams.get(param) && param !== 'district') {
```

Comment:
🟠 that param check looks to specific for this seem-to-be-generic hook

Reply:
it's related to this pr https://github.com/keenethics/mayak/pull/71

Reply:
refer to this PR [MID-69](https://trello.com/c/fViJdFP2/69-%D1%8E%D0%B7%D0%B5%D1%80-%D1%80%D0%BE%D0%B7%D0%B4%D1%96%D0%BB-%D1%81%D0%BF%D0%B5%D1%86%D1%96%D0%B0%D0%BB%D1%96%D1%81%D1%82%D0%B8-fe)